### PR TITLE
releasing App 0.6

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

prepare for the v0.6 release

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [ ] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

File: src/lightning_app/__version__.py [hunk 1]
```diff
@@ -1 +1 @@
-version = "0.6.0rc0"
```
Comment (carmocca) [inline src/lightning_app/__version__.py:1]: Isn't https://github.com/Lightning-AI/lightning/releases/tag/app%2F0.6.0rc0 already released?

Comment (Borda) [reply src/lightning_app/__version__.py:1]: no, we have pushed just RC

**Review by awaelchli — APPROVED** [review]

File: src/lightning_app/CHANGELOG.md [hunk 2]
```diff
@@ -5,19 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
-## [0.7.0] - 2022-MM-DD
+## [0.6.0] - 2022-09-06
```
Comment (awaelchli) [inline src/lightning_app/CHANGELOG.md:8]:
```suggestion
## [0.6.0] - 2022-09-07
```

```suggestion
## [0.6.0] - 2022-09-08
```

**Review by manskx — APPROVED** [review]

**Review by lantiga — APPROVED** [review]
Looks good. Let’s get S3 Drives out in short order as well.

**Pull request merged**